wallettemplate/build.gradle

```diff
@@ -1,12 +1,10 @@
 plugins {
-    id 'org.openjfx.javafxplugin' version '0.0.7' apply false
```

Comment:
This change looks good to me. To be consistent across all subprojects, could you change it for the other subprojects too?